### PR TITLE
[FE][SYNC-222] New ArticleCard Design

### PR DESCRIPTION
**Pull request by newb1er**
## Changes:
- Add new badges .svg files
- New components: `HashtagPill`
- Refactor `ArticleCard`
## Build Status:
✅ 
## Screenshots (optional)
image

### CODE REVIEW

File: src/components/ArticleCard.vue [hunk 1]
```diff
@@ -1,231 +1,169 @@
...
-.subscribe-btn {
-  position: absolute;
-  top: 1rem;
-  right: 1rem;
-  background: transparent;
-  border: 0;
-  padding: 0;
-  margin: 0;
+  &__lg {
+    width: 392px;
```
Comment (andyrochi) [inline src/components/ArticleCard.vue:161]: Which design does the `__lg` class correspond to? Or does this large attribute correspond to the profile page card?

File: src/components/ArticleCard.vue [hunk 2]
```diff
@@ -1,231 +1,169 @@
...
 
-<style lang="scss" scoped>
-@import '@/assets/scss/news.scss';
+<style lang="scss" >
+.article-card {
+  position: relative;
+  flex: 1;
+  width: 320px;
+  max-height: 388px;
+  min-width: 304px;
```
Comment (andyrochi) [inline src/components/ArticleCard.vue:101]: The design here seems to have changed.

File: src/components/ArticleCard.vue [hunk 3]
```diff
@@ -1,231 +1,169 @@
...
     },
-    bookmarkTooltip() {
-      return this.isSubscribed ? '取消收藏文章' : '收藏文章'
+    hasTags() {
+      return this.tags.length > 0
     },
-    cols() {
-      return this.full ? 12 : 4
+    hasCate() {
+      return this.category !== null
```
Comment (andyrochi) [inline src/components/ArticleCard.vue:78]: `hasCat` looks like a more intuitive naming scheme for me XD

File: src/components/ArticleCard.vue [hunk 4]
```diff
@@ -1,231 +1,169 @@
...
-            {{ getDateTime(lastUpdatedAt) }}
-          </div>
-          <div>觀看數：{{ viewsCount }} | 編輯數：{{ editedCount }}</div>
-        </div>
-      </template>
-    </b-card>
-  </b-col>
+  <div class="article-card" :class="{'article-card__lg':lg}" @click="GotoArticle">
+    <div class="article-card__image">
+      <b-img-lazy :src="imgSrc" fluid-grow />
```
Comment (andyrochi) [inline src/components/ArticleCard.vue:4]: image
The left section needs to be implemented too.
The title of each block should be displayed.

File: src/components/ArticleCard.vue (hunk 1 above)
Comment (newb1er) [reply src/components/ArticleCard.vue:161]: Corresponding to the large card design in large viewport.

**Comment by david20571015** [comment]
The layout of hashtag page is needed to be fix. 
image

**Comment by yccodr** [comment]
> The layout of hashtag page is needed to be fixed. image

The layout should be set in view instead of the component.

File: src/components/ArticleCard.vue (hunk 2 above)
Comment (yccodr) [reply src/components/ArticleCard.vue]: Removed. `width` is fixed.

**Pull request merged**